docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -565,6 +565,9 @@ Property name                                      Description
                                                    specification to use for new tables; either ``1`` or ``2``.
                                                    Defaults to ``2``. Version ``2`` is required for row level deletes.
 
+``metadata_compression_codec``                     Metadata compression codec;
+                                                   either ``NONE`` or ``GZIP``.  Defaults to ``NONE``.
```

Comment:
Under what circumstances a user would want to set this?

cc @danielcweeks @rdblue @alexjo2144